CODE_OF_CONDUCT.md

```diff
@@ -55,7 +55,7 @@ project may be further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at contact@fusetools.com.
+reported by contacting the project team at fuse-open@googlegroups.com.
```

Comment:
Actually, this one should be fuse-open@fusetools.com. It shouldn't be a publicly readable place, as reports may contain sensitive information.

Reply:
Fixed!